core/src/test/kotlin/fr/sncf/osrd/utils/DummyInfra.kt

```diff
@@ -25,43 +26,31 @@ class DummyInfra : RawInfra, BlockInfra {
 
     /** get the FullInfra  */
     fun fullInfra(): FullInfra {
+        // si je fais loadedSignalInfra = loadedSignalInfrasimulator.loadSignals(this)
+        // et ensuite je modifie les blocks, ça va être à jour dans loadedSignalInfra ?
```

Comment:
I don't think so.

But the signaling simulator could be instantiated once and be kept as a class field.

Reply:
made it a class field